packages/addons/service/net-snmp/changelog.txt

```diff
@@ -0,0 +1,3 @@
+8.0.100
```

Comment:
Please make this `100` then future backports from 9.0.100 (master) to 8.0.100 (LE8.0) are simple